pyschism/forcing/hycom/hycom2schism.py

```diff
@@ -715,6 +715,42 @@ def fetch_data(self, outdir: Union[str, os.PathLike], hgrid, vgrid, start_date,
             dst_temp['time'][it] = it
             temp_int = interp_to_points_3d(dep, y2, x2, bxyz, ptemp)
             temp_int = temp_int.reshape(zcor2.shape)
+            timeseries_t[it,:,:,0]=temp_int
```

Comment:
I am not sure about how this part became a delta, I don't recall making any changes to this part...